google/cloud/storage/internal/curl_download_request.cc

```diff
@@ -98,23 +95,29 @@ StatusOr<HttpResponse> CurlDownloadRequest::Close() {
 }
 
 StatusOr<ReadSourceResult> CurlDownloadRequest::Read(char* buf, std::size_t n) {
+  GCP_LOG(WARNING) << __func__ << "()";
```

Comment:
@devjgm hilariously, removing this like breaks the test.  Considering this is a single-threaded program I am starting to think that we should stop using computers.